client/my-sites/site-monitoring/components/site-logs-table/index.tsx

```diff
@@ -13,25 +13,37 @@ type SiteLogs = SiteLogsData[ 'logs' ];
 
 interface SiteLogsTableProps {
 	logs?: SiteLogs;
-	logType?: string;
+	currentPageIndex?: number;
+	logType?: LogType;
 	isLoading?: boolean;
+	headerTitles: string[];
+}
+
+export function formatColumnName( column: string ) {
+	if ( column === 'request_type' ) {
+		return 'Request type';
+	} else if ( column === 'request_url' ) {
+		return 'Request URL';
```

Comment:
Needs localization

Reply:
@danielbachhuber I was debating what to do with this - currently, we are not localizing them and they are automatically generated as a part of a function. I have some of these strings hard-coded because I needed to replace the spelling for the table headers but it does not have all the headers. If we need localization, we should hard code all the table headers

Reply:
Done